applications/luci-app-keepalived/htdocs/luci-static/resources/view/keepalived/script.js

```diff
@@ -0,0 +1,106 @@
+'use strict';
+'require view';
+'require ui';
+'require form';
+'require uci';
+
+return view.extend({
+	load: function() {
```

Comment:
Do we need this load function?

Reply:
Yes, If `uci.load('keepalived')` is not used in `load`, it does not list values from `uci.sections()`